definitions/linux.yaml

```diff
@@ -128,8 +128,8 @@ sources:
 - type: ARTIFACT
   attributes:
     names:
-      - UsersShellConfigs
-      - RootUserShellConfigs
+      - UsersShellHistory
+      - RootUserShellHistory
 labels: [Configuration Files]
```

Comment:
Is this still correct "[Configuration Files]" ? Seeing that these are now history files?